Sources/RInAppMessaging/Router.swift

```diff
@@ -43,7 +43,7 @@ internal protocol RouterType: ErrorReportable {
 /// Handles all the displaying logic of the SDK.
 internal class Router: RouterType, ViewListenerObserver {
 
-    private enum UIConstants {
+    enum UIConstants {
```

Comment:
please make it private again

Reply:
Updated here [57d4646](https://github.com/rakutentech/ios-inappmessaging/pull/275/commits/57d464617318d1c37bbc0e36be3d5a49f81e4b83).